python_modules/dagster/dagster/_core/definitions/antlr_asset_selection/AssetSelection.g4

```diff
@@ -0,0 +1,85 @@
+grammar AssetSelection;
```

Comment:
would suggest (1) putting all this generated stuff in a subfolder (2) splitting out the generated code into its own PR (3) clearly marking which files are autogenerated, either with a README in the folder or notes at the top of the files